model_analyzer/record/record.py

```diff
@@ -121,6 +122,18 @@ def aggregation_function():
 
         return (lambda records: max(records, key=lambda r: r.value()))
 
+    @staticmethod
+    def value_function():
+        """
+        The function that is used to combine records from multiple
```

Comment:
We need clear description to distinguish between aggregation_function. what is different?  That one shows that it takes a list of records and returns a single record.  This takes a list of records(?) and returns a float/int?

Reply:
Updated the comment and used the lambda function